frontend/management/commands/dummy_commands.py

```diff
@@ -0,0 +1,52 @@
+from frontend.models import (
```

Comment:
Please untrack this file for the PR: `git rm --cached frontend/management/commands/dummy_commands.py`

Reply:
Done!